nest/nest.py

```diff
@@ -504,6 +508,261 @@ def away_temperature(self, value):
         self._set('device', data)
 
 
+class ProtectDevice(NestBase):
+    @property
+    def _device(self):
+        return self._nest_api._status['topaz'][self._serial]
+
+    @property
+    def _topaz(self):
+        return self._nest_api._status['topaz'][self._serial]
```

Comment:
You are correct. I started out with topaz and did not realize I left it at the end when its not needed. Gonna make the change right away.